test/compiler/parsing/fixtures/watf/instruction/nested/actual.wast

```diff
@@ -0,0 +1,12 @@
+(func
+  (drop (i32.const 4))
+  (drop (block $t (br 1 (i32.const 8)) (i32.const 1)))
+
+  (drop
+    (block (result i32)
+      (drop (i32.const 4))
+      (drop (block $a (br 1 (i32.const 8)) (i32.const 1)))
```

Comment:
Note that this doesn't work with specific instruction like `br_if` amt, I still need to refactor a bit.